src/bun.js/modules/EventsModule.h

```diff
@@ -0,0 +1,14 @@
+#include "../bindings/ZigGlobalObject.h"
+#include "JavaScriptCore/JSGlobalObject.h"
+
+namespace Zig {
+
+inline void generateEventsSourceCode(JSC::JSGlobalObject* lexicalGlobalObject, JSC::Identifier moduleKey, Vector<JSC::Identifier, 4>& exportNames, JSC::MarkedArgumentBuffer& exportValues) {
+    JSC::VM& vm = lexicalGlobalObject->vm();
+    GlobalObject* globalObject = reinterpret_cast<GlobalObject*>(lexicalGlobalObject);
+
+    exportNames.append(JSC::Identifier::fromString(vm, "EventEmitter"_s));
```

Comment:
We need to export both the class but also the properties of the class

```
<ref *1> [Function: EventEmitter] {
  once: [AsyncFunction: once],
  on: [Function: on],
  getEventListeners: [Function: getEventListeners],
  EventEmitter: [Circular *1],
  usingDomains: false,
  captureRejectionSymbol: Symbol(nodejs.rejection),
  captureRejections: [Getter/Setter],
  EventEmitterAsyncResource: [Getter],
  errorMonitor: Symbol(events.errorMonitor),
  defaultMaxListeners: [Getter/Setter],
  setMaxListeners: [Function (anonymous)],
  init: [Function (anonymous)],
  listenerCount: [Function (anonymous)]
}
```

<img width="345" alt="image" src="https://user-images.githubusercontent.com/709451/185779422-955e535e-d144-421d-bc20-5a14cbcdf6a0.png">

Reply:
but maybe its fine if its the default export too